src/common/error.ts

```diff
@@ -50,3 +50,5 @@ export class InvalidContractConfigurationError extends BaseError {
     super(INVALID_CONTRACT_CONFIGURATION_ERROR);
   }
 }
+
+export class EvaluationTimeoutError extends BaseError {}
```

Comment:
where is this being thrown?

Reply:
in the query function

Reply:
It's not actually an evaluation issue, it's an aborted request. So we should throw an AbortError